assets/js/modules/adsense/components/setup/SetupMain.js

```diff
@@ -98,14 +98,17 @@ export default function SetupMain( { finishSetup } ) {
 	// Get additional information to determine account and site status.
 	const alerts = useSelect( ( select ) => select( STORE_NAME ).getAlerts( accountID ) );
 	const urlChannels = useSelect( ( select ) => select( STORE_NAME ).getURLChannels( accountID, clientID ) );
-	const error = useSelect( ( select ) => select( STORE_NAME ).getError() );
+	const accountsError = useSelect( ( select ) => select( STORE_NAME ).getError( 'getAccounts', [] ) );
+	const alertsError = useSelect( ( select ) => select( STORE_NAME ).getError( 'getAlerts', [ accountID ] ) );
+	const error = accountsError || alertsError;
```

Comment:
The IB says that `error` would be removed in this component, but it looks like perhaps we could still use this for cases where it is still used where it only seems to care "if there is any error". In that case, I think it would be slightly better to name it something like `hasError` and cast it to a boolean, but that's a bit of a nitpick.

I'm okay with keeping it if @felixarntz is.

Reply:
In principle, keeping `error` here like this is fine, however it's not fully accurate: With the new logic, we're looking up the two specific "expected errors" that are relevant for the `determineAccountStatus` function. However, right now any other "real error" won't be considered for the actual error rendering component further below.

Instead of this `error` variable, I think it would make sense to rely on the new `hasErrors` selector and write that into a `hasErrors` variable. This covers _any_ possible error in the AdSense store, similar to how it behaved before, but using the new selector from #1814.